modules/apps/list-type/list-type-test/src/testIntegration/java/com/liferay/list/type/service/test/ListTypeEntryServiceTest.java

```diff
@@ -231,10 +233,16 @@ private void _testUpdateListTypeEntry(User user) throws Exception {
 
 			listTypeEntry = _addListTypeEntry(user);
 
+			String externalReferenceCode = RandomTestUtil.randomString();
+
 			listTypeEntry = _listTypeEntryService.updateListTypeEntry(
-				listTypeEntry.getListTypeEntryId(),
+				externalReferenceCode, listTypeEntry.getListTypeEntryId(),
 				Collections.singletonMap(
 					LocaleUtil.US, RandomTestUtil.randomString()));
+
+			Assert.assertEquals(
```

Comment:
When creating the integration test in your next task, please remember my comment in Selton's pull request:

"I saw that we don't have an update test in ListTypeEntryLocalServiceTest, so instead of change this class, can you create a new test there? this way we would be able to also cover/assert the other fields values in our test, like name changes.
We have some examples in ObjectDefinitionLocalServiceTest and ObjectFieldLocalServiceTest."

Reply:
Sure, I won't forget about it!